templates/compose/cloudreve.yaml

```
@@ -0,0 +1,57 @@
+# documentation: https://docs.cloudreve.org/
+# slogan: A self-hosted file management and sharing system.
+# category: storage
+# tags: file sharing,cloud storage,self-hosted,open-source
+# logo: svgs/cloudreve.svg
+# port: 5212
+
+services:
+  cloudreve:
+    image: cloudreve/cloudreve:latest
+    depends_on:
+      cloudreve-db:
+        condition: service_healthy
+      cloudreve-redis:
+        condition: service_healthy
+    environment:
+      - SERVICE_URL_CLOUDREVE_5212
+      - CR_CONF_Database.Type=postgres
+      - CR_CONF_Database.Host=cloudreve-db
+      - CR_CONF_Database.User=${SERVICE_USER_POSTGRESQL}
+      - CR_CONF_Database.Password=${SERVICE_PASSWORD_POSTGRESQL}
+      - CR_CONF_Database.Name=${POSTGRES_DB:-cloudreve}
+      - CR_CONF_Database.Port=5432
+      - CR_CONF_Redis.Server=cloudreve-redis:6379
+      - CR_CONF_Redis.Password=${SERVICE_PASSWORD_REDIS}
+    volumes:
+      - backend_data:/cloudreve/data
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "5212"]
+      interval: 1m30s
```

Is there any specific reason why the interval is set to too long?

```suggestion
      interval: 20s
```